src/v2/features/characters/components/team-graph.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { ResponsiveLine } from '@nivo/line';
+
+export const TeamGraph: React.FC<{ data: { id: string; data: { x: string; y: number }[] }[] }> = ({ data }) => {
+    return (
+        <div style={{ height: '380px', width: '480px' }}>
+            <ResponsiveLine
+                data={data}
+                enablePoints={false}
+                enableArea={true}
+                colors={{ scheme: 'spectral' }}
+                lineWidth={1}
+                curve="stepAfter"
+                margin={{
+                    top: 10,
+                    right: 0,
+                    bottom: 10,
+                    left: 100,
+                }}
+                enableGridX={true}
+                axisBottom={null}
+                enableGridY={true}
+                axisLeft={{
+                    tickValues: [851, 2212, 5097, 11194, 21930, 40000],
+                }}
+                yScale={{
+                    type: 'linear',
+                    reverse: false,
+                    min: 0,
+                    max: 40000,
+                }}
+                gridYValues={[851, 2212, 5097, 11194, 21930, 40000]}
+                useMesh={true}
+                animate={false}
+            />
+        </div>
```

Comment:
Wrap into dialog component with icon 
you can use `battlefield-info.tsx` as example
![image](https://github.com/svehera/tacticusplanner/assets/33064347/c955abba-f148-4d7d-a26a-5e787ff000bd)

Reply:
I agree that this looks better.  However, the size of the chart and the size of the dialog are a bit of a problem:

`...
            <Dialog open={open} onClose={handleClose} maxWidth={isMobile ? 'xl' : 'lg'} fullWidth>
            <DialogTitle>Roster Power Distribution <InfoTeamGraphBox/></DialogTitle>
            <DialogContent>
                <div style={{ height: '380px' }}>
...`

The graph does not show unless a height is defined (and setting it to a percentage doesn't work), I'm not sure how this will look on a mobile device (short of testing on mobile after pushing this to a real test instance).